pages/common/dolt-init.md

```diff
@@ -0,0 +1,12 @@
+# dolt init
+
+> Create an empty Dolt data repository.
```

Comment:
The 2nd example isn't an empty repo, right?
```suggestion
> Initialize a new Dolt data repository.
```

Reply:
I took this sentence from the official documentation.

the init commit push metadata only.

Solved in 2b940581aa22cd07b8b4b2703dd5e3daabeb8b4a